src/kOS/Suffixed/BodyTarget.cs

```diff
@@ -96,6 +96,7 @@ private void BodyInitializeSuffixes()
             AddSuffix("ROTATIONPERIOD", new Suffix<double>(() => Body.rotationPeriod));
             AddSuffix("ATM", new Suffix<BodyAtmosphere>(() => new BodyAtmosphere(Body)));
             AddSuffix("ANGULARVEL", new Suffix<Vector>(() => RawAngularVelFromRelative(Body.angularVelocity)));
+            AddSuffix("SOIRADIUS", new Suffix<double>(() => Body.sphereOfInfluence));
```

Comment:
Well.
1. It complies
2. it runs. 
3. I can even print SOI radius on the screen. (see screenshot) 
4. There are no warnings about potential problems in the [KSP API documentation](http://wiki.kerbalspaceprogram.com/wiki/API:CelestialBody)
P.S. what exactly do you mean by "not super reliable"?